### PR TITLE
Cleaned up ROADMAP.md

### PR DESCRIPTION
**Pull request by captainsafia**
That word wrap, tho.

### CODE REVIEW

**Comment by jdfreder** [comment]
Thanks @captainsafia ! 🐸 🍰

File: ROADMAP.md [hunk 1]
```diff
@@ -1,29 +1,42 @@
...
-A big part of the notebook is the ecosystem of libraries that rely on it. We need to make sure that we support many of the common libraries in use across Python, Julia, R, and JavaScript. Libraries to ensure support of:
+* Packaging and Installation
+    * OS X and Linux packaged release
+* Specification Compliance
+    * Read compliance with Notebook format v3 (v4 is already supported)
+* Editor Functionality
+    * LaTeX (https://github.com/nteract/mathjax-electron)
+    * Code completion and introspection
+    * Clipboard
+    * Undo/redo stack
```
Comment (rgbkrk) [inline ROADMAP.md:30]: The undo redo stack would also be for cell operations as well, really anything user initiated on the document.

**Comment by rgbkrk** [comment]
Thanks Safia.

**Pull request merged**